src/commands/moderation/createrole.js

```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, Constants, Color } = require('../../')
+
+module.exports = class CreateRole extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'createrole',
+      category: 'moderation',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_ROLES'], permissions: ['MANAGE_ROLES'] },
+      parameters: [
+        { type: 'color', required: false },
+        { type: 'string', full: true, missingError: 'commands:createrole.noParams', required: true }
+      ]
+    })
+  }
+
+  async run ({ channel, guild, author, t }, color = new Color('#ffffff'), name) {
+    const hexcode = color.rgb(true)
+    const embed = new SwitchbladeEmbed(author)
+
+    try {
+      await guild.createRole({ name, color: hexcode })
+      embed
+        .setTitle(t('commands:createrole.successTitle'))
+        .setDescription(t('commands:createrole.successMessage', { name }))
+        .setColor(hexcode)
+    } catch (err) {
+      embed
```

Comment:
Shouldn't we throw a `CommandError` instead?

Reply:
Are you sure?? We would want the guild using the bot to know that something didn't go well, not the host of the bot.

Reply:
`CommandError`s actually send error embeds instead of throwing console errors. Search around the repo and check it out

Reply:
Ahh I see! Cool! Fixed now.